iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/DanglingLineXml.java

```diff
@@ -132,4 +161,30 @@ protected void readSubElements(DanglingLine dl, NetworkXmlReaderContext context)
             }
         });
     }
+
+    private static void readBoundaryPoint(String dlId, BoundaryPoint boundaryPoint, NetworkXmlReaderContext context) {
+        XMLStreamReader reader = context.getReader();
+        double p = XmlUtil.readOptionalDoubleAttribute(reader, "p");
+        double q = XmlUtil.readOptionalDoubleAttribute(reader, "q");
+        double v = XmlUtil.readOptionalDoubleAttribute(reader, "v");
+        double angle = XmlUtil.readOptionalDoubleAttribute(reader, ANGLE);
+        context.getEndTasks().add(() -> {
+            checkBoundaryValue(p, boundaryPoint.getP(), "p", dlId);
+            checkBoundaryValue(q, boundaryPoint.getQ(), "q", dlId);
+            checkBoundaryValue(v, boundaryPoint.getV(), "v", dlId);
+            checkBoundaryValue(angle, boundaryPoint.getAngle(), ANGLE, dlId);
+        });
+    }
+
+    private static void checkBoundaryValue(double imported, double calculated, String name, String dlId) {
+        if (!Double.isNaN(imported) && imported != calculated) {
+            LOG.info("boundaryPoint.{} of DanglingLine {} is recalculated. Its imported value is not used (imported value = {}; calculated value = {})", name, dlId, imported, calculated);
+        }
+    }
+
+    private static boolean hasDefinedBoundaryPoint(DanglingLine dl) {
+        BoundaryPoint boundaryPoint = dl.getBoundaryPoint();
+        return !Double.isNaN(boundaryPoint.getP()) || !Double.isNaN(boundaryPoint.getQ()) ||
+                !Double.isNaN(boundaryPoint.getV()) || !Double.isNaN(boundaryPoint.getAngle());
+    }
```

Comment:
If values are computed, why do we read/write them? For instance we do not serialize the intensity (Terminal.getI())

Reply:
Good question, I don't remember why we wanted to serialize it... maybe it is not useful as you said. What do you think @annetill ?

Reply:
After thinking and @zamarrenolm 's input, I think you are right